ui/pages/send/send-footer/send-footer.component.test.js

```diff
@@ -30,7 +30,7 @@ describe('SendFooter Component', () => {
       <SendFooter
         addToAddressBookIfNew={propsMethodSpies.addToAddressBookIfNew}
         amount="mockAmount"
-        clearSend={propsMethodSpies.clearSend}
```

Comment:
there were two methods that did the same overall thing, `clearSend` and `resetSendState`. narrowed this usage to `resetSendState` which is the more descriptive of the two.